src/components/Ruler/index.js

```diff
@@ -40,12 +40,14 @@ const calculateY = (points, yTooltipPosition) => {
 };
 
 const Ruler = ({ ruler, points, height, width }) => {
-  const xTooltipMargin = 5;
+  const timeLabelMargin = 5;
   // The fixed position of a x-axis label which is the same for all highlighted points
-  const xTooltipPosition = height - labelHeight - xTooltipMargin;
-  const pointsObject = calculateY(points, xTooltipPosition);
+  const timeTooltipPosition =
+    typeof ruler.getTimeLabelPosition === 'function'
```

Comment:
Does this need to be added to the propTypes?

Reply:
Yes, I will add it.